clusterloader2/pkg/measurement/common/network-policy/manifests/policy-egress-allow-apiserver.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{.Name}}
+  namespace: {{.Namespace}}
+  labels:
+    test: {{.TestClientLabel}}
+spec:
+  podSelector:
+    matchLabels:
+      test: {{.TestClientLabel}}
+  policyTypes:
+  - Egress
+  egress:
+  - ports:
+    - port: 443
+      protocol: TCP
+    - port: 80
+      protocol: TCP
+    to:
+    - ipBlock:
+        cidr: {{.kubeAPIServerIP}}/32
```

Comment:
Probably you could use something like:
```
  egress:
    - toEntities:
        - kube-apiserver
 ```
instead (https://docs.cilium.io/en/v1.12/policy/language/#entities-based). But it's minor, non-blocking.

Reply:
`toEntities` and `fromEntities` would work only for Cilium implementation I think.
I don’t see these fields as part of K8s network policy API.
I think we should keep it as is.

Reply:
ah yea, I totally forgot it's NetworkPolicy and not CiliumNetworkPolicy.